[#73443] 17.3 Workflows changes  https://community.openproject.org/wo… (#22586)
opf/openproject: docs/system-admin-guide/manage-work-packages/work-package-workflows/README.md

Select the type of work package for which you want to edit the workflow, e.g. *Task*. 

Once opened, you can configure workflows for this type:

1. Choose whether you want to edit default transitions, or transitions when a user is the **author** or **assignee** using the tabs at the top of the page.

![Tabs to select between default transitions, when the user is the author or when the user is the assignee](admin_workflow_tabs.png)

2. Select the **role** for which you want to configure the workflow. The workflow table will update automatically when switching roles.

3. Define which **statuses** are available for this type:
   - Click **+ Status** to add or remove statuses.
   - Select the statuses you want to associate with this type and apply your changes.
   - Removing a status will make it unavailable for this type and delete existing workflow transitions for it.
   - Newly added statuses will appear in the workflow table immediately and can be configured before saving.

> [!NOTE]
> If a status has no transitions configured, it will be removed automatically when saving.

4. Configure the allowed status transitions in the workflow table:

>    - The matrix shows the **current status on the Y axis (rows)** and the **new status on the X axis (columns)**.

Reviewer: Can we just say rows and columns here? Mixing those and the X/Y axes is confusing imo

Author: can do.

>    - The matrix shows the **current status on the Y axis (rows)** and the **new status on the X axis (columns)**.

Author: ```suggestion
   - The matrix shows the **current status in the rows** and the **new status in the columns**.
```
